data-service/src/main/java/uk/gov/gchq/palisade/service/data/web/DataController.java

```diff
@@ -28,42 +25,39 @@
 import org.springframework.web.bind.annotation.RestController;
 import org.springframework.web.servlet.mvc.method.annotation.StreamingResponseBody;
 
-import uk.gov.gchq.palisade.exception.ForbiddenException;
-import uk.gov.gchq.palisade.reader.request.DataReaderRequest;
-import uk.gov.gchq.palisade.service.data.config.StdSerialiserConfiguration;
-import uk.gov.gchq.palisade.service.data.config.StdSerialiserPrepopulationFactory;
-import uk.gov.gchq.palisade.service.data.model.AuditSuccessMessage;
+import uk.gov.gchq.palisade.service.data.model.AuditErrorMessage;
+import uk.gov.gchq.palisade.service.data.model.AuditableAuthorisedDataRequest;
+import uk.gov.gchq.palisade.service.data.model.AuditableDataResponse;
 import uk.gov.gchq.palisade.service.data.model.DataRequest;
-import uk.gov.gchq.palisade.service.data.service.AuditService;
-import uk.gov.gchq.palisade.service.data.service.DataService;
+import uk.gov.gchq.palisade.service.data.model.TokenMessagePair;
+import uk.gov.gchq.palisade.service.data.service.AuditMessageService;
+import uk.gov.gchq.palisade.service.data.service.AuditableDataService;
 
 import java.io.OutputStream;
-import java.util.concurrent.atomic.AtomicLong;
 
+/**
+ * Controller for data-service.  Provides the front end RESTFul web service for resources that have already been
+ * registered with the Palisade Service.  The request is in the form of information that will uniquely identify the
+ * resource request and will return a data stream of the response data.
+ */
 @RestController
 @RequestMapping(path = "/")
```

Comment:
our other services are under /api, migrate tests and controller to use /api/...

Reply:
still outstanding

Reply:
This one has gone around the block ("/" -> "/api -> "/"data" -> "/").  The last word from Adam is that this one should be path = "/".